src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -98,10 +98,14 @@ public static DynamoPreferencesData Default()
     /// </summary>
     public struct HostAnalyticsInfo
     {
-        // Dynamo variation identified by host.
+        // Dynamo variation identified by host, e.g. Dynamo Revit
         public string HostName;
         // Dynamo variation version specific to host
         public Version HostVersion;
+        // Dynamo host application name, e.g. Revit
+        public string HostAppName;
```

Comment:
maybe this should be `HostProcessName`? It's not immediately clear to me (and I would guess others) what the difference is between this and `HostName` - process name is very clear in my mind.

Reply:
It's not exactly the process name since e.g. AutoCAD process name is ACAD, year the naming of this particular one is tricky

Reply:
How about `HostProductName`  and `HostProductVersion`?